### DISCUSSION TITLE
Does this support remote GUI editing?

### DISCUSSION THREAD

**Discussion by kajweb** [mockoon/mockoon#573] [General]
My A and B computers connect to the C server for remote development through VS code.
But only computers A and B can access service D. I want to run mockoon on my C server.
How can I edit the rules through the C server and initiate a request by computers A and B in Proxy mode?

**Comment by 255kb** [comment]
For Proxy mode, Mockoon is usually sitting between your computer (A or B) and the proxied server (D). If Mockoon can't access it (D) there is no other way to forward the incoming trafic. 
I think one solution could be to create a redirection or tunnel from A or B to the server D, and setup Mockoon proxy mode with A or B's tunnel address.

**Reply by kajweb** [reply]
If computers A and B are in different locations，How can I synchronize the configuration of A and B computers?

**Comment by kajweb** [comment]
Just like Computer A is in the company and Computer B is at home.
I want to continue the configuration file that has been set by computer A on computer B after get off work.

**Reply by 255kb** [reply]
There is currently no synchronization feature in Mockoon. However, we are going to release, today or tomorrow, a new version (v1.16.0) with a revamped storage system. In this version, you will be able to move your data files anywhere on your hard drive and setup a Git repository. This will definitely make sharing the settings between computers easier.
